shopyo/manage.py

```diff
@@ -1,136 +1,49 @@
-import json
-import os
-import shutil
-import subprocess
 import sys
 
 from flask_migrate import Migrate, MigrateCommand
 from flask_script import Manager
 
-from addon import db
+from shopyoapi.init import db
 from app import app
-from initialise import add_admin, add_setting
+
+from shopyoapi.cmd import new_project
+from shopyoapi.cmd import clean
+from shopyoapi.cmd import initialise
 
 migrate = Migrate(app, db, compare_type=True)
 manager = Manager(app)
 
 manager.add_command("db", MigrateCommand)
 
-with open("config.json", "r") as config:
-    config = json.load(config)
-# @manager.command
-def initialise():
-
-    print("Creating Db")
-    print("#######################")
-    subprocess.run([sys.executable, "manage.py", "db", "init"], stdout=subprocess.PIPE)
-    print("Migrating")
-    print("#######################")
-    subprocess.run(
-        [sys.executable, "manage.py", "db", "migrate"], stdout=subprocess.PIPE
-    )
-    subprocess.run(
-        [sys.executable, "manage.py", "db", "upgrade"], stdout=subprocess.PIPE
-    )
-
-    print("Initialising User")
-    print("#######################")
-    add_admin(
-        config["user"]["id"],
-        config["user"]["name"],
-        config["user"]["password"],
-        config["user"]["admin"],
-    )
 
-    print("Initialising Settings")
-    print("#######################")
-    for name, value in config["settings"].items():
-        add_setting(name, value)
-    print("Done!")
-
-
-#@manager.command
+# @manager.command
 def runserver():
     app.run()
 
 
-#@manager.command
+# @manager.command
```

Comment:
@Abdur-rahmaanJ I noticed these had been commented out, but they could be removed completely right?

Reply:
ah yes yes